README.rst

```diff
@@ -25,36 +25,36 @@ The repository consists mostly of externally hosted subrepositories:
 +------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------+
 | `flac <https://github.com/xiph/flac>`_                                       | `1.4.3 with cross-compilation patch <https://github.com/xiph/flac/releases/tag/1.4.3>`_                           | `BSD-3 license (+ GPL for utils, not used by DALI) <https://github.com/xiph/flac/blob/master/COPYING.Xiph>`_        |
 +------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------+
-| `opus <https://github.com/xiph/opus>`_                                       | `1.4 <https://github.com/xiph/opus/releases/tag/v1.4>`_                                                           | `BSD-3 license <https://github.com/xiph/opus/blob/master/COPYING>`_                                                 |
+| `opus <https://github.com/xiph/opus>`_                                       | `1.5.2 <https://github.com/xiph/opus/releases/tag/v1.5.2>`_                                                           | `BSD-3 license <https://github.com/xiph/opus/blob/master/COPYING>`_                                                 |
 +------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------+
-| `FFmpeg <https://github.com/FFmpeg/FFmpeg>`_                                 | `7.0.2 <https://github.com/FFmpeg/FFmpeg/releases/tag/n7.0.2>`_                                                   | `LGPL v2.1 license <https://github.com/FFmpeg/FFmpeg/blob/master/LICENSE.md>`_                                      |
+| `FFmpeg <https://github.com/FFmpeg/FFmpeg>`_                                 | `7.1 <https://github.com/FFmpeg/FFmpeg/releases/tag/n7.1>`_                                                   | `LGPL v2.1 license <https://github.com/FFmpeg/FFmpeg/blob/master/LICENSE.md>`_                                      |
```

Comment:
If we need to revert, please remember to revert it here as well.

Reply:
Done